src/ResourceManager/LocationBasedServices/Commands.LocationBasedServices/LocationBasedServicesAccount/NewAzureLocationBasedServicesAccountKey.cs

```diff
@@ -0,0 +1,102 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.LocationBasedServices.Properties;
+using Microsoft.Azure.Commands.LocationBasedServices.Models;
+using Microsoft.Azure.Management.LocationBasedServices;
+using Microsoft.Azure.Management.LocationBasedServices.Models;
+using System.Globalization;
+using System.Management.Automation;
+using LocationBasedServicesModels = Microsoft.Azure.Management.LocationBasedServices.Models;
+
+namespace Microsoft.Azure.Commands.LocationBasedServices
+{
+    /// <summary>
+    /// Regnerate Location Based Services Account Key (Primary or Secondary)
+    /// </summary>
```

Comment:
I would advise against ConfirmImpact.High for this cmdlet.  The mechanism for skipping this is less than ideal (either increaing the ConfirmPreference or setting ```-Confirm:$false```  on the command line.  It is better in these circumstances to use SHouldContinue/Force on all calls instead, but the advice is generally to implement SHouldProcess, and assme the user knows the impact (and can use -Confirm or -WhatIf if they are not sure).

Reply:
ok will remove.